### PR TITLE
Issue 1144

### PR DESCRIPTION
**Pull request by jollopre**
Create account org is needed now. Fixed find nil whne swaping orgs.

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -125,13 +125,14 @@ def identifier_for(scheme)
   # @param new_organisation_id [Integer] the id for an organisation
   # @return [String] the empty string as a causality of setting api_token
   def org_id=(new_org_id)
+    puts "$$$$ new_org_id: #{new_org_id}"
```
Comment (briri) [inline app/models/user.rb]: remove this debug statement

File: app/models/user.rb [hunk 2]
```diff
@@ -125,13 +125,14 @@ def identifier_for(scheme)
...
     unless self.can_change_org? || new_org_id.nil? || self.org.nil? || (new_org_id.to_s == self.org.id.to_s)
       # rip all permissions from the user
       self.perms.delete_all
     end
     # set the user's new organisation
     super(new_org_id)
-    self.save!
+    # self.save!
```
Comment (briri) [inline app/models/user.rb]: delete the line. we have git ;)

File: lib/assets/javascripts/views/shared/create_account_form.js [hunk 3]
```diff
@@ -1,25 +1,22 @@
 import initAutoComplete from '../../utils/autoComplete';
 import ariatiseForm from '../../utils/ariatiseForm';
 import { togglisePasswords } from '../../utils/passwordHelper';
-import { SHOW_SELECT_ORG_MESSAGE, SHOW_OTHER_ORG_MESSAGE } from '../../constants';
```
Comment (briri) [inline lib/assets/javascripts/views/shared/create_account_form.js]: We should remove those constants from the file if we don't need them anymore

File: app/models/user.rb (hunk 2 above)
Comment (jollopre) [reply app/models/user.rb]: :)

**Pull request merged**